meson.build

```diff
@@ -53,6 +53,11 @@ endif
 if not cy.version().version_compare('>=0.29.35')
   error('SciPy requires Cython >= 0.29.35')
 endif
+# NOTE: Cython tends to do beta releases
+# that may cause this check to choke
```

Comment:
If that's a risk, let's not add this? The constraint in `pyproject.toml` should be okay.